lib/Client.php

```diff
@@ -49,7 +49,7 @@ public function getDefaultTimeout()
 
     /**
      * @param RequestInterface $apiRequest
-     * @return \stdClass
+     * @return mixed
```

Comment:
@williamokano this method should only return an `stdClass` object. Why the `mixed` declaration instead?